tests/unit/main.fmf

```diff
@@ -40,6 +40,11 @@ require+:
     /basic:
         summary: Basic unit tests (development packages)
         tier: 0
+        adjust+:
+          - check:
+              - how: avc
+                result: xfail
+            because: https://bugzilla.redhat.com/show_bug.cgi?id=2342247
```

Comment:
Would it make sense to apply this change only when running on rawhide?

Reply:
Yeah, I guess it's a rawhide container, generating AVC denials host machine logs and tmt discovers? is that why we see them on other distros too?